spotify_player/src/event/page.rs

```diff
@@ -194,7 +194,7 @@ fn handle_key_sequence_for_search_page(
                 CommandOrAction::Command(command) => {
                     window::handle_command_for_artist_list_window(command, artists, &data, ui)
                 }
-                CommandOrAction::Action(action) => {
+                CommandOrAction::Action(action, _) => {
```

Comment:
should this be

```suggestion
                CommandOrAction::Action(action, ActionTarget::SelectedItem) => {
```
?

Same with other matches in this file

Reply:
You're right, this was an oversight on my part